driver/sql/projector_stream.go

```diff
@@ -25,7 +25,8 @@ type StreamProjector struct {
 
 // NewStreamProjector creates a new projector for a projection
 func NewStreamProjector(
-	db *sql.DB,
+	rdb *sql.DB,
+	wdb *sql.DB,
```

Comment:
Change the naming of the variables to be more explicit or include a better documentation in the comments.